Automattic/woocommerce-payments
# Add payout and paydown transactions to the transactions list

Fixes #3458

![image](https://user-images.githubusercontent.com/3295/151369876-61346c47-6bad-4b6f-8f1a-9ecb3a528c25.png)

#### Changes proposed in this Pull Request

This PR makes those modifications:

- Formats the transaction list rows by showing "N/A" and "-" [like it's shown in the Figma](https://www.figma.com/file/v71ana8m4qMOkawTyQonDS?node-id=53:3186).
- Rephrases "Financing paydown" and "Financing payout" as "Loan repayment" and "Loan dispersement"
- Makes the "Loan dispersement" row unclickable
- Adds filter for the "Loan dispersement" and "Loan repayment" transaction types in Advanced > Type filter.
- Adds filter for filtering related rows with a loan ID in Advanced > Loan filter.
- Removes the modifications when Capital feature flag isn't active
 
#### Testing instructions

* Ensure you've checked out 1547-gh-Automattic/woocommerce-payments-server on your server, or it is merged before starting the client testing.
* Clear your cache to get the loan info in the account cache, if you don't see it.
* Prepare a loan that has payout and pay down transactions if you didn't have it before. 
* Open WP-Admin > Payments > Transactions page
* Look at the differences when you have Capital enabled/disabled. Check the filters exist when Capital is enabled, and absent when not.
* Check if the payout transaction is clickable.
* Check if the pay down transactions open the related charge transaction. (The timeline item is another issue, so it's not present in this PR)
* If you have multiple loans Check if the Loan filters work. 
* Check if the pay down and payout type filters work.
* In the Loan filter, check if you see "In Progress" on the active loan option, and on the inactive loans, you see "Paid at full"
* Check if the design matches the Figma prototype.

-------------------


**Post merge**
